docs/UserGuide.md

```diff
@@ -150,556 +157,1428 @@ An example of the final command is displayed below.
+
+Syntax: `help`
+
+### 2.9.2 Uploading a csv file
+
+Uploading of a valid csv file into Trackr allows Trackr to parse the data to `Task`, `Order`, `Supplier` and `Menu Items` and adds them to their respective lists.
+
+It is important to strictly follow the format specified below for your data to parse successfully.
 
-Examples:
+<div markdown="block" class="alert alert-tip">
+
+:information_source: **Information**
 
-* `tab t/HOME` switches the tab to the `Home` tab
+**Step 1:** Start each row with the type of item you want to add (i.e. `Orders`)
 
-## Others
+**Step 2:** List out on the next row all the respective item fields (i.e. `OrderName`, `Quantity`, ...)
 
-### Uploading a csv file
+**Step 3:** Fill in the next few rows will all the corresponding data, without leaving any fields empty (i.e. `Chocolate Cookies`, `10`, ...)
 
-Uploads a valid csv file onto Trackr and parses each add command for `Task`, `Order` and `Suppliers`, and adds them to their respective lists. It is important that a strict format is used in order for the commands to be parsed correctly.
+Note: For any **optional** fields, you can indicate it with a `-` to have it be taken as its default value.
 
-* First indicate what item is to be added. (i.e. `Orders`)
-* List out the item fields in the next row. (i.e. `OrderName`, `Quanity`, `Deadline`, etc.)
-* Fill in the valid values for each field. (i.e. `Chocolate Cookies`, `10`, `10/01/2023`, etc.)
-* If there are any optional fields (i.e. `Suppler`'s `Tag` field), indicate it with a '-' if you don't wish to put any values in that particular field.
+</div>
 
 <p align="center">
-  <img src="images/ValidCsvFile.png" width="600" />
-  <br>Figure 3: Valid csv file Example
+  <img src="images/ug/ValidCsvFile.png"/>
+  <br>Figure 2.9.2.1: Example of a valid csv file
 </p>
 
-### Exiting the program: `exit`
+### 2.9.3 Exiting the program: `exit`
 
 Exits the program.
 
 Syntax: `exit`
 
-# Data
+# 3 Data
+
+This section will inform you how the data is saved.
 
-### Saving the data
+## 3.1 Saving of data
 
-Save changes after any command executed successfully. There is no need to save manually.
+All data are saved after any command is executed successfully. There is no need for you to manually save.
 
-### Editing the data file
+## 3.2 Editing the data file
 
-Trackr data are saved as a JSON file `[JAR file location]/data/trackr.json`. Advanced users are welcome to update data directly by editing that data file.
+Trackr data are saved as a JSON file at `[JAR file location]/data/trackr.json`.
+
+Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">
-:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, Trackr will discard all data and start with an empty data file at the next run.
+
+:warning: **Caution:**<br><br>
+
+It is advised to make your changes through our application and not through editing the data file directly.
```

Comment:
We might be able to assume that the user will not touch the code of the product so we should be able to remove this section of the UG. What do you think?